MWT/architecture.rst

```diff
@@ -0,0 +1,99 @@
+Architecture
+============
+
+Structure
+---------
+
+.. figure:: images/hierarchy.png
+   :alt: MWT Hierarchy
+   :align: center
+
+Widget
+~~~~~~
+
+A widget is an object that is intended to be displayed on a screen and that can interact with the user. A Widget occupies a specific region of the display and holds a state.
```

Comment:
the user interacts with the widget, not the opposite i think

Reply:
Ok, fixed.